.github/workflows/test-e2e-pr.yaml

```diff
@@ -38,18 +36,9 @@ jobs:
       - name: Setup K3d
         uses: ./.github/actions/k3d
 
-      - name: Login to GHCR
```

Comment:
do we not need this for the nightly?

Reply:
good catch, added to nightly + `GITHUB_TOKEN` env var